code/utils.py

```diff
@@ -18,7 +23,20 @@ def reportAccuracy(true_labels, predicted_labels, label_dict):
     # label_dict is a 15x1 cell array where each entry is a string
     # containing the name of that category
     # accuracy is a scalar, defined in the spec (in %)
-
+    
```

Comment:
I know label_dict was provided by the starter code, but do you know for sure if we don't need it? true_labels and predicted_labels are both integer arrays, but do the ints correspond with the label